app/src/main/java/com/github/sdpteam15/polyevents/fragments/LoginFragment.kt

```diff
@@ -98,6 +97,7 @@ class LoginFragment : Fragment() {
                 if (newValue!!) {
                     if (inDbObservable.value!!) {
                         //If already in database redirect
+                            // TODO: pass retrieved user value to the profile fragment to pass the user info
```

Comment:
I think it's not necessary to get the user information since the fragment will get the info on its own

Reply:
I see thanks!